pkg/skaffold/build/local/bazel.go

```diff
@@ -41,6 +41,7 @@ func (b *Builder) buildBazel(ctx context.Context, out io.Writer, workspace strin
 	args = append(args, a.BazelArtifact.BuildArgs...)
 	args = append(args, a.BazelArtifact.BuildTarget)
 
+	// FIXME: is it possible to apply b.skipTests?
```

Comment:
FWIW, I think it would be possible to run a separate query to retrieve the test targets from the top level build directory, and then skip those in the args we pass to `bazel build`.